### PR TITLE
103 named routes constants

### LINKED ISSUE #103: use constants for named routes

### PR DESCRIPTION
**Pull request by tamari-gray**
fixes #103

### CODE REVIEW

File: lib/navigation/route_names.dart [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const String waiting_for_game_to_start = '/waiting_for_game_to_start';
```
Comment (nickmeinhold) [inline lib/navigation/route_names.dart]: Following up on the "pollute the namespace" conversation, you could consider putting routeName constants inside the relevant classes, eg. 

```Dart
class CreateGameScreen2 extends StatelessWidget {
  static const routeName = '/create_game2';
...
```

and then in the MaterialApp routes:

```Dart
CreateGameScreen2.routeName: (context) => CreateGameScreen2(),
```

**Review by nickmeinhold — CHANGES_REQUESTED** [review]
Looks great mate - I did "Request changes" just because we talked about it and I know you planned to make the changes.  Please feel free to hit that Squash & Merge when you feel she's ready!

File: lib/navigation/route_names.dart (hunk 1 above)
Comment (tamari-gray) [reply lib/navigation/route_names.dart]: beautiful

**Comment by tamari-gray** [comment]
squashiiiing!

**Pull request merged**